From eeeac4e9ba8a6513428b456a551bffd95d533e50 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@amd.com>
Date: Sun, 26 Feb 2023 08:54:11 -0600
Subject: [PATCH 5/5] trivial: only run device tests if directory exists

---
 data/installed-tests/fwupd.sh | 22 ++++++++++++++--------
 1 file changed, 14 insertions(+), 8 deletions(-)

diff --git a/data/installed-tests/fwupd.sh b/data/installed-tests/fwupd.sh
index ae553c5c0..4fec27740 100755
--- a/data/installed-tests/fwupd.sh
+++ b/data/installed-tests/fwupd.sh
@@ -10,6 +10,19 @@ run_test()
         fi
 }
 
+run_device_tests()
+{
+	if [ -d @devicetestdir@ ]; then
+		# grab device tests from the CDN to avoid incrementing the download counter
+		export FWUPD_DEVICE_TESTS_BASE_URI=http://cdn.fwupd.org/downloads
+		for f in `grep --files-with-matches -r emulation-url @devicetestdir@`; do
+		        echo "Emulating for $f"
+		        fwupdmgr device-emulate --no-unreported-check --no-remote-check --no-metadata-check "$f"
+		        rc=$?; if [ $rc != 0 ]; then exit $rc; fi
+		done
+	fi
+}
+
 run_test acpi-dmar-self-test
 run_test acpi-facp-self-test
 run_test acpi-phat-self-test
@@ -25,14 +38,7 @@ run_test uefi-dbx-self-test
 run_test synaptics-prometheus-self-test
 run_test dfu-self-test
 run_test mtd-self-test
-
-# grab device tests from the CDN to avoid incrementing the download counter
-export FWUPD_DEVICE_TESTS_BASE_URI=http://cdn.fwupd.org/downloads
-for f in `grep --files-with-matches -r emulation-url @devicetestdir@`; do
-        echo "Emulating for $f"
-        fwupdmgr device-emulate --no-unreported-check --no-remote-check --no-metadata-check "$f"
-        rc=$?; if [ $rc != 0 ]; then exit $rc; fi
-done
+run_device_tests
 
 # success!
 exit 0
-- 
2.34.1

